packages/ui/src/ReactMarkdown.tsx

```diff
@@ -86,36 +91,42 @@ const MarkdownLink: (navigateNote: NavigateNote) => React.FC<any> =
         );
     };
 
-const flatten = (text, child): any =>
+const flatten = (text: String, child: ElementContent): String =>
```

Comment:
wird type string groß geschrieben?

Reply:
hups. it worked, but seems it is not the same as string ^^' https://stackoverflow.com/questions/14727044/what-is-the-difference-between-types-string-and-string using string seems to be the correct way.